akka-contrib/src/main/scala/akka/contrib/pattern/DistributedPubSubMediator.scala

```diff
@@ -548,6 +574,10 @@ class DistributedPubSubMediator(
       content = bucket.content + (key -> ValueHolder(v, valueOption))))
   }
 
+  def getCurrentTopics(): Map[String, ActorRef] = {
+    context.children.map(ref ⇒ (URLDecoder.decode(ref.path.name, "utf-8"), ref)).toMap
+  }
```

Comment:
@BPoserow hmm, I don't know the reason for that. Some kind of wrong classpath issue. Please push the change to this branch and I will try to run it to understand better.